pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2024,6 +2024,37 @@ var _ = Describe("Converter", func() {
 			Entry("use virtio transitional", true),
 			Entry("use virtio non-transitional", false),
 		)
+		DescribeTable("Should set the error policy", func(epolicy v1.DiskErrorPolicy, expected string, specified bool) {
+			vmi.Spec.Domain.Devices.Disks[0] = v1.Disk{
+				Name: "mydisk",
+				DiskDevice: v1.DiskDevice{
+					Disk: &v1.DiskTarget{
+						Bus: v1.VirtIO,
+					},
+				},
+			}
+			if specified {
+				vmi.Spec.Domain.Devices.Disks[0].ErrorPolicy = epolicy
+			}
+			vmi.Spec.Volumes = append(vmi.Spec.Volumes, v1.Volume{
+				Name: "mydisk",
+				VolumeSource: v1.VolumeSource{
+					Ephemeral: &v1.EphemeralVolumeSource{
+						PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
+							ClaimName: "testclaim",
+						},
+					},
+				},
+			})
+			domainSpec := vmiToDomainXMLToDomainSpec(vmi, c)
+			Expect(string(domainSpec.Devices.Disks[0].Driver.ErrorPolicy)).To(Equal(expected))
+		},
+			Entry("ErrorPolicy not specified", nil, "stop", false),
```

Comment:
^ FWIW I think it was the use of `nil` here that threw me. I still don't think we need to supply both a value and bool here.

Reply:
Yes, now I removed it and using a pointer instead